docs/UserGuide.md

```diff
@@ -230,7 +247,125 @@ _Written by: Brian Wong Yun Long_
 
 ## Budgeting and Financial Insights
 
-### Viewing the Statistics: `stats`
+### Viewing Moolah Statistics: `stats`
+View the general statistics of monetary transactions, from 3 main views: Categorical Savings, Monthly Expenditure and
+Time Insights. Various statistics listed include total income, total expense, total savings, total expenses, sum of expenses
+ordered by category or time and more.
+
+
+#### Statistics for Categorical Savings
+View the total amount sum of all transactions in each category. 
+
+**Format:** `stats s/categorical_savings`
```

Comment:
That's a great idea to split the stats into different parts, it makes `stats` command so much easier to understand!